content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -259,6 +259,36 @@ For more information, see "[`github context`](/actions/reference/context-and-exp
 **Required** The shell where you want to run the command. You can use any of the shells listed [here](/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell). Required if `run` is set.
 {% endif %}
 
+#### `runs.steps[*].if`
+
+**Optional** You can use the `if` conditional to prevent a step from running unless a condition is met. You can use any supported context and expression to create a conditional.
+
+{% data reusables.github-actions.expression-syntax-if %} For more information, see "[Expressions](/actions/learn-github-actions/expressions)."
+
+**Example: Using contexts**
+
+ This step only runs when the event type is a `pull_request` and the event action is `unassigned`.
+
+ ```yaml
+steps:
+  - name: My first step
+    if: {% raw %}${{ github.event_name == 'pull_request' && github.event.action == 'unassigned' }}{% endraw %}
+    run: echo This event is a pull request that had an assignee removed.
+```
+
+**Example: Using status check functions**
+
+The `my backup step` only runs when the previous step of a composite action fails. For more information, see "[Expressions](/actions/learn-github-actions/expressions#job-status-check-functions)."
```

Comment:
Might be worth adding a footnote or something to this [page](https://docs.github.com/en/actions/learn-github-actions/expressions#job-status-check-functions) to indicate that in composite actions these refer to the success/failure of the composite action not the job

Reply:
I agree. Should we also distinguish between  composite `step-if` (depends on composite action's success)  from `pre-if` and `post-if` (depends on job's success)?

What do you think about this:

Rename `Job Status Check Expressions` to `Status Check Expressions`, explain that these expressions can be deconstructed using `job.status` in a `job.step` and `github.action_status` in a `composite.step`, respectively, and show some examples like you mentioned: `if: ${{ github.action_status == 'failure' }}`

Reply:
Sounds good to me

Reply:
If a user is using a third party action, will it be obvious to them whether the action is a composite action?

Or does this apply to [all small-a actions](https://docs.github.com/en/actions/creating-actions/about-custom-actions#types-of-actions) (not just composite actions)?

Reply:
@skedwards88 This only applies to composite actions, other actions can't have conditional steps.

The idea is that maintainers of the composite action should only have to consider the action itself being successful, not the job executing it.

For example, let's say a job is failing, but its last step calls a composite action with `if:always()`. We want this composite action to start out with a state of `success` and track changes to its state independent of the job.